distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -990,9 +990,12 @@ type Table
        - case_sensitivity: Specifies if the text values should be compared case
          sensitively.
        - on_problems: Specifies the behavior when a problem occurs.
-    tokenize_to_rows : Text | Integer -> Text -> Case_Sensitivity -> Table
-    tokenize_to_rows self column pattern="." case_sensitivity=Case_Sensitivity.Sensitive =
-        Split_Tokenize.tokenize_to_rows self column pattern case_sensitivity
+       - at_least_one: If True, a tokenization that returns no values will still
+         produce at least one row, with Nothing for the output column values.
+         Equivalent to converting a tokenization output of [] to [Nothing].
```

Comment:
I'd like to think about the argument name. `at_least_one` sounds a bit unclear to me.

Maybe `keep_empty_results` could be clearer?

Or at least, `at_least_one_row`? I know that the method has `_rows` in it, but IMO the argument without the `_row` suffix makes it unclear _at least one_ of *_what_*?

Reply:
Using at_least_one_row.